.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,35 @@
+## Motivation
+Add references to relevant tickets or a short description about what motivated you do it. (E.g JIRA: https://issues.jboss.org/browse/AEROGEAR-{} AND/OR ISSUE: https://github.com/aerogear/standards/issues/{}) 
```

Comment:
@camilamacedo86 
Can we change the GH issues url with https://github.com/aerogear/ios-showcase-template/{}

Reply:
Sure 👍